app/src/utils/btcsnap-signer.ts

```diff
@@ -81,15 +84,19 @@ export class BtcSnapSigner implements RemoteSigner {
 
   async getPublicKey(): Promise<string> {
     const network = await this.getNetwork();
+    if (network === bitcoin) {
+      await updateNetworkInSnap(BitcoinNetwork.Test)
```

Comment:
Is this a typo, or on purpose to force testnet for the demo?

It looks odd to query the snap in the previous statement (inside `this.getNetwork()`) to then set the network in the snap. I'm not sure what the intention here is.

Reply:
Removed I just had it to force the network